src/utils/date.ts

```diff
@@ -0,0 +1,25 @@
+import { DateTime } from "luxon";
+import logger from "../lib/Logger";
+
+export const toReadableFormat = (dateToConvert: string | undefined, lang = "en"): string => {
+    if (!dateToConvert) {
+        return "";
+    }
+    const jsDate = new Date(dateToConvert);
+    const dateTime = DateTime.fromJSDate(jsDate);
+    let convertedDate;
+    switch (lang) {
```

Comment:
May be clearer as if/else

```suggestion
if (lang) {
  convertedDate = dateTime.setLocale(lang).toFormat("d MMMM yyyy");
}
else {
  convertedDate = dateTime.setLocale("en").toFormat("d MMMM yyyy");
}
```

Reply:
changed to if / else